CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1884,14 +1845,12 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float l
       for (j = 0; j < TXSIZE; ++j) {
         //  Take next largest x-slice for the x-template (it reduces bias in the forward direction after irradiation)
         //		   xtemp_[i][j+2]=(1.f - xxratio)*thePixelTemp_[index_id_].entx[imaxx][ilow].xtemp[i][j] + xxratio*thePixelTemp_[index_id_].entx[imaxx][ihigh].xtemp[i][j];
-        //		   xtemp_[i][j+2]=(1.f - xxratio)*thePixelTemp_[index_id_].entx[iyhigh][ilow].xtemp[i][j] + xxratio*thePixelTemp_[index_id_].entx[iyhigh][ihigh].xtemp[i][j];
-        if (flip_x) {
+        //		   xtemp_[i][j+2]=(1.f - xxratio)*entx20_->xtemp[i][j] + xxratio*entx22_->xtemp[i][j];
```

Comment:
Is this commented out code needed: add one line explaining its usage, if so; otherwise just remove

Reply:
I left this one, I think the comment above explains its usage